applications/example/test/http.test.js

```diff
@@ -62,7 +52,8 @@ const getRequest = task => {
 
 config.tasks.forEach(task => {
   const name = task.get || task.post;
-  metatests.test('http request of ' + name, test => {
+  test.endAfterSubtests();
+  test.test('http request of ' + name, test => {
     const request = getRequest(task);
     if (!request.path) {
       test.bailout();
```

Comment:
While we are at it, AFAIK `test.bailout` works badly in Impress app failing the whole worker and hanging the tests indefinitely (because of runner.on('finish') never getting called due to such failure). I'd suggest replacing all of them with test.error at least for now.

/cc @tshemsedinov